db/migrate/20170421154535_create_default_price_groups_for_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDefaultPriceGroupsForUsers < ActiveRecord::Migration
+  def up
+    User.find_each do |user|
+      # does nothing if user_based_price_groups is not turned on
+      user.create_default_price_group!
+    end
+  end
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
This whole feature feels like a fairly high-risk change, so I'd like to be confident we could roll back if necessary.